Makefile

```diff
@@ -7,8 +7,14 @@ check-clean-cache: error
 test: error
 	go test ./...
 
-error: errorutil
+error: build-errorutil
 	./errorutil -d . update
 
 errorutil:
-	go run github.com/layer5io/meshkit/cmd/errorutil -d . update
\ No newline at end of file
+	go run github.com/layer5io/meshkit/cmd/errorutil -d . update
+
+errorutil-analyze:
+	go run github.com/layer5io/meshkit/cmd/errorutil -d . analyze
+
+build-errorutil:
```

Comment:
@mgfeller I think we can get rid of line 10:11 because we have line 13:14. What do you think?

Reply:
@kumarabd not quite sure, the purpose is to run the locally built binary for testing purposes just in this repo when developing the utility, in a branch. I assume `go run github.com/layer5io/meshkit/cmd/errorutil` reflects the latest remote version. So I think for that use case, `errorutil` is not equivalent to `error`. But it could be removed anyway, if you want.

Reply:
No problem, sure keep it doesn't harm anything. Lets go with this PR.